### ISSUE TITLE
90.1 PRM: Redundant SpaceType Lights schedule created for each Space in SpaceType

### ISSUE THREAD

**Issue by eringold** [NREL/openstudio-standards#1527]
@dmaddoxwhite @lymereJ @weilixu 

I'm getting multiple redundant lighting schedules created in the stable PRM baseline model, it looks like one per instance of Space in each SpaceType:
![image](https://github.com/NREL/openstudio-standards/assets/10799524/9367b40c-1062-4fb7-a29f-1107d1a12114)

I think this is due to the logic in ashrae_90_1_prm.SpaceType `space_type_light_sch_change`. For each space, the SpaceType lights schedule is retrieved, then the name changed, then it checks whether the _new_ name is in `schedule_hash`, and if not the schedule is copied and assigned again to the Lights, and the new name added to `schedule_hash`. Because the name is changed before it's checked, the `else` branch will run and a new copy will always happen for each space:

https://github.com/NREL/openstudio-standards/blob/12bbfabf3962af05b8c267c1da54b8e3a89217a0/lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.SpaceType.rb#L440-L468

Basically I think https://github.com/NREL/openstudio-standards/blob/12bbfabf3962af05b8c267c1da54b8e3a89217a0/lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.SpaceType.rb#L453

should be 
```
if schedule_hash.key?(ltg_schedule_name)
```

**Comment by weilixu** [comment]
@eringold thanks for reporting. This issue will be addressed in the next release.
